pages/trip/[id].js

```diff
@@ -24,19 +24,27 @@ export default function TripDetailsPage() {
   }
 
   const { title, location, timePeriod, img } = trips;
+
   const startDate = timePeriod.map((timePeriod) => `${timePeriod.startDate}`);
   const endDate = timePeriod.map((timePeriod) => `${timePeriod.endDate}`);
 
+  const longitude = location.map((longitude) => `${longitude.longitude_city}`);
+  const latitude = location.map((latitude) => `${latitude.latitude_city}`);
+
   return (
-    <TripDetails
-      title={title}
-      location={location}
-      handleDelete={handleDelete}
-      img={img}
-      timePeriod={timePeriod}
-      id={id}
-      startDate={startDate}
-      endDate={endDate}
-    />
+    <>
+      <TripDetails
+        title={title}
+        location={location}
+        handleDelete={handleDelete}
+        img={img}
+        timePeriod={timePeriod}
+        id={id}
+        startDate={startDate}
+        endDate={endDate}
+        latitude={latitude}
+        longitude={longitude}
+      />
+    </>
```

Comment:
The Fragment` <>...</>` is redundant since there's only a single child component, TripDetails, being returned. ✨

```suggestion
    <>
      <TripDetails
        title={title}
        location={location}
        handleDelete={handleDelete}
        img={img}
        timePeriod={timePeriod}
        id={id}
        startDate={startDate}
        endDate={endDate}
        latitude={latitude}
        longitude={longitude}
      />
    </>
```